src/Symfony/Component/Cache/CacheItem.php

```diff
@@ -100,6 +100,44 @@ public function expiresAfter($time)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function tag($tag)
+    {
+        $tag = self::normalizeTag($tag);
+        $this->tags[$tag] = $tag;
+
+        return $this;
+    }
+
+    /**
+     * Validates a cache tag.
```

Comment:
This could be updated because the method not only validates but also modifies/normalizes the tags.